foundation/foundation-archetype/src/main/resources/archetype-resources/.mvn/maven-build-cache-config.xml

```diff
@@ -43,9 +43,9 @@
                 <dirName>dockerbuild</dirName>
                 <dirName>krausening</dirName>
                 <!-- Reports needed on CI to ensure consistent reporting of QA metrics: -->
-                <dirName>cucumber-reports</dirName>
-                <dirName>surefire-reports</dirName>
-                <dirName>failsafe-reports</dirName>
```

Comment:
Q: why is this?  For `dist`, I think it was b/c it doesn't natively look in that folder.  But I would have expected this to be fine in `target`.

Reply:
The config file assumes the build directory as the default, which is set to `dist` in habushu. The relative pathing traverses out of the dist directory to look for the target directory.